core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
As you wish. I really don't mind.

If you have the time to give it a final polish and do a PR, please do:

1. styles probably need to go into vanilla/base
2. the buttons probably need tooltips (similar to AdvancedSearch)
3. the PR should point to a demo to show what changed / how it works